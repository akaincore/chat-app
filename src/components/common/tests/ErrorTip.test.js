/* eslint-env jest */
import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ErrorTip from '../ErrorTip';

const mockProps = {
  error: null,
};

describe('<ErrorTip />', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ErrorTip {...mockProps} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });
  it('renders correctly', () => {
    const tree = renderer.create(<ErrorTip {...mockProps} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
