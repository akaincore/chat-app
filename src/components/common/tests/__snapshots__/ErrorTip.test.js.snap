// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorTip /> renders correctly 1`] = `null`;
